src/actions/actions.js

```diff
@@ -0,0 +1,56 @@
+const databaseName = 'todoz'
+const databaseName_test = 'todoz_test'
+let connectionString
+
+if(process.env.NODE_ENV === 'development'){
+  connectionString = `postgres://${process.env.USER}@localhost:5432/${databaseName}`
+} else if (process.env.NODE_ENV === 'test'){
+  connectionString = `postgres://${process.env.USER}@localhost:5432/${databaseName_test}`
+}
+
+// const pgp = require('pg-promise')();
+// const db = pgp( connectionString );
+const db = require('../database/db.js')
+
+
+const getAllTodos = () => {
+  return db.any(`SELECT id, description FROM todos
+    WHERE is_completed = false`)
+}
+
+
+const getCompletedTodos = () => {
+  return db.any(`SELECT * FROM
+    todos WHERE is_completed = true`)
+
+}
+
+
+const completeOneTodo = (id) => {
+  return db.any(`UPDATE todos
```

Comment:
make this a `db.one` since I can't see an instance where we should expect more than one row be returned.